packages/health-check/src/checks/critical.js

```diff
@@ -60,7 +86,44 @@ async function uploadCheck(done) {
     data.ip = error?.response?.ip ?? null;
   }
 
-  done({ name: "upload_file", time: calculateElapsedTime(time), ...data });
+  done({ name, time: calculateElapsedTime(time), ...data });
+}
+
+// skylinkHealthCheck checks if the skylink has reached full redundancy
+async function skylinkHealthCheck(skylink, numRetries = 30, authCookie, isLarge = false) {
+  // Get the health of the skylink
+  const response = await got(`https://${process.env.PORTAL_DOMAIN}/skynet/health/skylink/${skylink}`, {
+    headers: { cookie: authCookie },
+  });
+  const healthData = getResponseContent(response);
+
+  // Check Basesectorredundancy first
+  if (healthData.basesectorredundancy !== 10 && numRetries > 0) {
+    // Semi-smart sleep before retrying. Sleep longer if the redundancy is
+    // lower.
+    await new Promise((r) => setTimeout(r, (10 - healthData.basesectorredundancy) * 1000));
+    return skylinkHealthCheck(skylink, numRetries - 1, authCookie, isLarge);
+  }
+
+  // Check the Fanout redundancy if it is a large file
+  if (isLarge && healthData.fanoutredundancy != 3 && numRetries > 0) {
```

Comment:
`nit` maybe `3` and `10` could be constants with a meaningful name like `DEFAULT_FANOUT_REDUNDANCY` or whatever

Reply:
is the redundancy configurable in skyd? should we pull the configuration values off of skyd and compare to those?

Reply:
it is configurable but we use the defaults for the portal and I can't think of a reason that we would ever not use the defaults on our portals.

applications might choose to use different defaults, but that doesn't apply here. 

I'll update to constants